mariadb/content.md

```diff
@@ -117,7 +117,13 @@ Set to a non-empty value, like `yes`, to generate a random initial password for
 
 ### `MARIADB_ROOT_HOST` / `MYSQL_ROOT_HOST`
 
-This is the hostname part of the root user created. By default this is `%`, however it can be set to any default [MariaDB allowed hostname component](https://mariadb.com/kb/en/create-user/#host-name-component).
+This is the hostname part of the root user created. By default this is `%`, however it can be set to any default [MariaDB allowed hostname component](https://mariadb.com/kb/en/create-user/#host-name-component). Setting this to `localhost` will prevent any root user being accessible except via the unix socket.
+
+### `MARIADB_MYSQL_LOCALHOST_USER` / `MARIADB_MYSQL_LOCALHOST_GRANTS`
```

Comment:
If you choose that `MARIADB_MYSQL_LOCALHOST_GRANTS` implies `MARIADB_MYSQL_LOCALHOST_USER` (see https://github.com/MariaDB/mariadb-docker/pull/409#pullrequestreview-873937215), please document it.